api/all/src/main/java/io/opentelemetry/api/trace/Span.java

```diff
@@ -274,7 +280,8 @@ default Span addEvent(String name, Attributes attributes, Instant timestamp) {
    * <p>Only the value of the last call will be recorded, and implementations are free to ignore
    * previous calls.
    *
-   * @param statusCode the {@link StatusCode} to set.
+   * @param statusCode the {@link StatusCode} to set. {@code null} is prohibited, and in that case
```

Comment:
Instead of these, can we generify the above even further `{@code null} is prohibited for any method argument in this class, and if provided, the behavior is undefined.`? Well maybe we'll add a `@Nullable` parameter later, but we may not either so we could address it later if that happens.

Reply:
It seems reasonable to me to put this up at the class level, yes.